lib/internal/bootstrap/node.js

```diff
@@ -137,7 +137,7 @@
       setupQueueMicrotask();
     }
 
-    if (process.binding('config').experimentalWorker) {
+    if (internalBinding('options').getOptions('--experimental-worker')) {
```

Comment:
can you factor out the `internalBinding` into a single call up top and also save the flag calls as variables to be used later?

something like 
```js
const { getOptions } = internalBinding('options');
const experimentalModules = getOptions('experimentalModules');
// ...
if (experimentalModules) {
  // ...
```

Reply:
Yes I will do that.
But one question. There is already some code in place like line 114:
`const options = internalBinding('options');
    if (options.getOptions('--help')) {
      NativeModule.require('internal/print_help').print(process.stdout);
      return;
    }`

Should I redo that part also and change code to use as mentioned:
`const { getOptions } = internalBinding('options');
const experimentalModules = getOptions('experimentalModules');
// ...
if (experimentalModules) {
  // ...`

Thanks

Reply:
As mentioned in my previous comment, will code like this (replacing line 114) be Ok:
```
    const { getOptions } = internalBinding('options');
    const helpOption = getOptions('--help');
    const completionBashOption = getOptions('--completion-bash');
    const experimentalModulesOption = getOptions('--experimental-modules');
    const experimentalVMModulesOption = getOptions('--experimental-vm-modules');
    const experimentalWorkerOption = getOptions('--experimental-worker');
```

Reply:
@burgerboydaddy Yes, that definitely looks okay :)